tests/rptest/scale_tests/tiered_storage_cache_stress_test.py

```diff
@@ -19,6 +19,8 @@
 import time
 from enum import Enum
 
+S3_ERROR_LOGS = ['unexpected REST API error "Internal Server Error" detected']
```

Comment:
how about we retry instead of ignoring it?

Reply:
It's retried by redpanda and the test passes, but when it retries it writes this error log which fails the CI run